### PR TITLE
Add connected organization and organization type for SSFs to the committee about tab

### LINKED ISSUE #3078: Add connected organization to SSF profile pages
**What we're after:**
Following on the [design work](https://github.com/fecgov/fec-cms/issues/3024), we need to now add connected organizations to those select committee types. Specifically, this new information will be added as the last item on the About this committee tab. 

### Related issues
- https://github.com/fecgov/fec-cms/issues/3024

### Completion criteria
- [ ] Add "Connected organization" to profile pages for committees with connected organizations. This means where org_type is: 
C corporation
L labor organization
M membership organization
T trade association
V cooperative
W corporation without capital stock

SSF examples:

org_tp | min | max
-- | -- | --
W | [C00012153](https://www.fec.gov/data/committee/C00012153/?tab=about-committee) | [C00690149](https://www.fec.gov/data/committee/C00690149/?tab=about-committee)
C | [C00000042](https://www.fec.gov/data/committee/C00000042/?tab=about-committee) | [C00709956](https://www.fec.gov/data/committee/C00709956/?tab=about-committee)
L | [C00000000](https://www.fec.gov/data/committee/C00000000/?tab=about-committee) | [C00702043](https://www.fec.gov/data/committee/C00702043/?tab=about-committee)
V | [C00000422](https://www.fec.gov/data/committee/C00000422/?tab=about-committee) | [C00681338](https://www.fec.gov/data/committee/C00681338/?tab=about-committee)
M | [C00000117](https://www.fec.gov/data/committee/C00000117/?tab=about-committee) | [C99002727](https://www.fec.gov/data/committee/C99002727/?tab=about-committee)
T | [C00000026](https://www.fec.gov/data/committee/C00000026/?tab=about-committee) | [C00702092](https://www.fec.gov/data/committee/C00702092/?tab=about-committee)

- [ ] For committees without name, leave blank. 

### Design
Screen Shot 2019-07-26 at 9 42 29 AM

### LINKED ISSUE #3079: Decide on the business rules for what org type information to show on PAC profile pages
**What we're after:**
Following up on the work to add connected organization names, we want to now make it clear which PAC type a user is looking at when on a committee profile page. It is somewhat unclear what the committee type is when it says PAC - Nonqualified. We think we can do better. This would also make these pages consistent with other profile pages such as candidate committee profile pages and parties.

### Related issues
- https://github.com/fecgov/fec-cms/issues/3078
- https://github.com/fecgov/fec-cms/issues/3024

### Completion criteria
- [ ] see @PaulClark2 to determine business rules for committee_type/org_type requirements
- [ ] Consult with content team for desired business logic on the appropriate language/descriptor for each org type beyond what's in the database

### Example: 
Screen Shot 2019-07-26 at 1 26 38 PM

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Resolves #3078 
- Resolves #3079  

Adds connected organization for organization types:

- C corporation
- L labor organization
- M membership organization
- T trade association
- V cooperative
- W corporation without capital stock

Adds organization type to the "Committee type" field if the attribute exists. The existence of organization type classifies the committee as an SSF.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Committee profile page - About this committee tab

## Screenshots
Example: http://localhost:8000/data/committee/C00702092/?tab=about-committee
Screen Shot 2019-09-12 at 3 43 56 PM

## How to test
- ./manage.py runserver
- Check any organization that is the following organization_type: C, L, M, T, V, and W has a connected committee on the page. Examples: 

org_tp | committee_id
-- | --
W | [C00690149](http://localhost:8000/data/committee/C00690149/?tab=about-committee)
C | [C00709956](http://localhost:8000/data/committee/C00709956/?tab=about-committee)
L | [C00702043](http://localhost:8000/data/committee/C00702043/?tab=about-committee)
V | [C00681338](http://localhost:8000/data/committee/C00681338/?tab=about-committee)
M | [C99002727](http://localhost:8000/data/committee/C99002727/?tab=about-committee)
T | [C00702092](http://localhost:8000/data/committee/C00702092/?tab=about-committee)

- Check other committees that are not the above org type. Such as candidate committees: [C00431445](http://localhost:8000/data/committee/C00431445/?tab=about-committee)
- Check that organization type is listed on the page if it exists. Example: [C00690149](http://localhost:8000/data/committee/C00690149/?tab=about-committee)

### CODE REVIEW

**Comment by lbeaufort** [comment]
@patphongs looks like that's an existing issue, if you'd like me to just put in a follow-up issue I think we could just add conditional logic for designation.

**Comment by lbeaufort** [comment]
I also found this one - I need to investigate whether this is an API issue or CMS issue: 

http://localhost:8000/data/committee/C00409011/

Screen Shot 2019-09-23 at 9 13 05 AM

**Comment by patphongs** [comment]
> looks like we need to handle when designation is missing - see http://localhost:8000/data/committee/C00621490/

@lbeaufort When committee type is None, should we not display it at all?

**Comment by lbeaufort** [comment]
@patphongs not displaying anything at all makes sense to me.

File: fec/data/templates/partials/committee/about-committee.jinja [hunk 1]
```diff
@@ -58,17 +58,24 @@
         </tr>
         <tr>
           <td class="figure__label">Committee type:</td>
-          <td class="figure__value">{{ committee_type_full }}</td>
+          <td class="figure__value">
+            {# If organizaton_type exists it's a SSF #}
```
Comment (lbeaufort) [inline fec/data/templates/partials/committee/about-committee.jinja]: Sometimes `organization_type` is H (convention) and I (inaugural) - see https://github.com/fecgov/openFEC/issues/3967. I think we can keep the logic as-is but maybe the comment is a little misleading.

Comment (lbeaufort) [reply fec/data/templates/partials/committee/about-committee.jinja]: @patphongs sorry for the confusion - I think we should only show org_type_full for the "real" SSF org types - we may want to put some sort of `is_ssf` flag in the view to take this logic out of the template and not repeat ourselves.

**Pull request merged**